src/sonic-yang-models/tests/yang_model_tests/tests/port.json

```diff
@@ -73,5 +73,13 @@
      "PORT_INVALID_MUX_CABLE_TEST": {
          "desc": "PORT_INVALID_MUX_CABLE_TEST non-boolean values, expect fail",
          "eStrKey": "InvalidValue"
+     },
+     "PORT_VALID_MULTIASIC_TEST": {
+         "desc": "PORT_VALID_MULTIASIC_TEST no failure."
+     },
+     "PORT_INVALID_MULTIASIC_TEST": {
```

Comment:
Is there a list of values that are considered to be invalid?

Reply:
Any string other than the ones added in "pattern" for role is considered to be invalid?

Reply:
Yes, string must match the pattern